agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/Global.java

```diff
@@ -8,23 +8,13 @@
     @Nullable
     private static volatile TelemetryClient telemetryClient;
 
-    private static volatile double samplingPercentage = 100;
-
     // this can be null if agent failed during startup
     @Nullable
     public static TelemetryClient getTelemetryClient() {
         return telemetryClient;
     }
 
-    public static double getSamplingPercentage() {
```

Comment:
I still think sampling rate inside TraceState is a bit awkward since you will need to strip it out before sending out the payload.  Can we keep it in Global? AiSampler can access Global, but then you may need to setDelegate inside AiSampler again.  Can we chat offline about this?